MADRaS/envs/gym_madras.py

```diff
@@ -15,9 +15,9 @@
 import math
 from copy import deepcopy
 import numpy as np
-import MADRaS.utils.snakeoil3_gym as snakeoil3
```

Comment:
Aah... could you give me a test that will fail without `MADRaS` in the import path? I want to make sure that the rest of the training/test scripts run smoothly because currently they don't when we have `MADRaS` in the import paths. 

PS. I haven't set `PYTHONPATH` to MADRaS root in my local.

Reply:
I added `MADRaS` in the path. Nothing breaks :)